esrally/chart_generator.py

```diff
@@ -1003,6 +1007,82 @@ def io(title, environment, race_config):
+        env_filter = TimeSeriesCharts.filter_string(environment, race_config)
+        vis_state = {
+            "title": title,
+            "type": "metrics",
+            "params": {
+                "axis_formatter": "number",
+                "axis_position": "left",
+                "id": str(uuid.uuid4()),
+                "index_pattern": "rally-results-*",
+                "interval": "1d",
+                "series": [
+                    {
+                        "axis_position": "left",
+                        "chart_type": "bar",
+                        "color": "#68BC00",
+                        "fill": "0.6",
+                        "formatter": "bytes",
+                        "id": str(uuid.uuid4()),
+                        "line_width": "1",
+                        "metrics": [{"id": str(uuid.uuid4()), "type": "sum", "field": "value.single"}],
+                        "point_size": 1,
+                        "seperate_axis": 1,
+                        "split_mode": "terms",
+                        "split_color_mode": "rainbow",
+                        "stacked": "stacked",
+                        "filter": "",
+                        "terms_size": "1000",
+                        "terms_order_by": "_key",
+                        "terms_direction": "asc",
+                        "terms_field": "field",
+                        "label": "Disk Usage",
+                        "value_template": "{{value}}",
+                        "steps": 0,
+                    }
+                ],
+                "show_legend": 1,
+                "show_grid": 1,
+                "drop_last_bucket": 0,
+                "time_field": "race-timestamp",
+                "type": "timeseries",
+                "filter": f"name:disk_usage_total {env_filter}",
+                "annotations": [
+                    {
+                        "fields": "message",
+                        "template": "{{message}}",
+                        "index_pattern": "rally-annotations",
+                        "query_string": f'((NOT _exists_:track) OR track:"{race_config.track}") AND ((NOT _exists_:chart) OR chart:io) '
+                        f'AND ((NOT _exists_:chart-name) OR chart-name:"{title}") AND environment:"{environment}"',
+                        "id": str(uuid.uuid4()),
+                        "color": "rgba(102,102,102,1)",
+                        "time_field": "race-timestamp",
+                        "icon": "fa-tag",
+                        "ignore_panel_filters": 1,
+                    }
+                ],
+                "axis_min": "0",
+            },
+            "aggs": [],
+            "listeners": {},
+        }
+
+        return {
+            "id": str(uuid.uuid4()),
+            "type": "visualization",
+            "attributes": {
+                "title": title,
+                "visState": json.dumps(vis_state),
+                "uiStateJSON": "{}",
+                "description": "io",
```

Comment:
It's in my mind to go through all these and make them more helpful. Since you should change this from `io` anyway, can you provide a short description here about the data/vis?

Reply:
I've added a super short description. I could do a longer thing if you'd like, but I'm kind of close to the project so my tiny blurb feels fine to me. Limited vision. If you have an idea for a better description I'd be happy to take it.